core-web/libs/portlets/dot-ema/src/lib/edit-ema-editor/components/edit-ema-palette/components/edit-ema-palette-content-type/edit-ema-palette-content-type.component.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { createComponentFactory, Spectator } from '@ngneat/spectator/jest';
+
+import { FormControl } from '@angular/forms';
+
+import { DotMessageService } from '@dotcms/data-access';
+
+import { EditEmaPaletteContentTypeComponent } from './edit-ema-palette-content-type.component';
+
+import { EditEmaPaletteStoreStatus } from '../../shared/edit-ema-palette.enums';
+import { CONTENT_TYPE_MOCK } from '../../shared/edit-ema-palette.mocks';
+
+describe('EditEmaPaletteContentTypeComponent', () => {
+    let spectator: Spectator<EditEmaPaletteContentTypeComponent>;
+
+    const createComponent = createComponentFactory({
+        component: EditEmaPaletteContentTypeComponent,
+        providers: [
+            {
+                provide: DotMessageService,
+                useValue: {
+                    get() {
+                        return 'Sample';
+                    }
+                }
+            }
+        ]
+    });
+
+    beforeEach(() => {
+        spectator = createComponent({
+            props: {
+                contentTypes: CONTENT_TYPE_MOCK,
+                filter: '',
+                control: new FormControl(''),
+                paletteStatus: EditEmaPaletteStoreStatus.LOADED
+            }
+        });
+    });
+
+    it('should emit dragStart event on drag start', () => {
+        const dragSpy = jest.spyOn(spectator.component.dragStart, 'emit');
+
+        spectator.triggerEventHandler('.content-type-card', 'dragstart', {
```

Comment:
Please don't use css classes for tests selector, use testId.

Reply:
Spectator dont accept byTestId() on triggerEventHandler, only selector/directives/type :(